proposals/0216-widget-support.md

```diff
@@ -323,8 +535,47 @@ After investigating impact to SDL Core, the impact is expected to be minor. Glob
 2. Changes to State Controller to allow HMI level transitions for widgets and screens
 3. Changes to Request Controller as RPCs from one app can be addressed to different screens
 
+T screen managers should be refactored to read screen capabilities notifications as well as the deprecated parameters.
+
 ## Alternatives considered
 
-To reduce complexity on the head unit, screen duplication can be supported by the SDL libraries. This could allow more flexibility to  display duplication for existing screens. However this increases the number of RPCs to be sent by the app and the user might see a delay in screen updates (duplication not synchronized).
+To reduce complexity on the head unit, screen duplication can be supported by the SDL libraries. This could allow more flexibility to display duplication for existing screens. However this increases the number of RPCs to be sent by the app and the user might see a delay in screen updates (duplication not synchronized).
```

Comment:
Update to:

To reduce complexity on the head unit, screen duplication can be supported by the SDL libraries. This could allow more flexibility to display duplication for existing screens. **However,** this increases the number of RPCs to be sent by the app and the user might see a delay in screen updates (duplication not **being** synchronized).